core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -261,6 +261,18 @@ sealed class NonEmptyMapOps[K, A](private[data] val value: NonEmptyMap[K, A]) {
     loop(head, tail).value
   }
 
+  def nonEmptyTraverse[G[_], L, B](
```

Comment:
I think this name isn't right. Traverse on maps only uses the values, not the keys. So if you are going to use the keys I think we need a new name. Something like `nonEmptyTraversePairs` or traverseWithKey or something.

Reply:
`NonEmptyMap` already has `map` (values only) and `mapBoth` (keys with values). So I guess this should be `nonEmptyTraverseBoth` then.